libobs/media-io/video-io.h

```diff
@@ -42,6 +42,7 @@ enum video_format {
 	VIDEO_FORMAT_YVYU,
 	VIDEO_FORMAT_YUY2, /* YUYV */
 	VIDEO_FORMAT_UYVY,
+	VIDEO_FORMAT_UYVA, /* two-plane, packed luma/chroma like UYVY and then alpha plane */
```

Comment:
This change breaks ABI. Please add the new format to the end of the enum.

Reply:
I didn't think of that.  Thank you